examples/complete/fixtures.us-east-2.tfvars

```diff
@@ -0,0 +1 @@
+example = "Hello world!"
```

Comment:
is this necessary? I feel like this file and the `variables.tf` in the complete example needs to be manually updated with the repo's `variables.tf`. It might be easier to put these values directly in the module reference, no ? This way we can kill the `variables.tf` file as well.

Reply:
The purpose is to setup the scaffolding to ensure contributors have these files. Do not want to kill the [`variables.tf` file as this is the terraform convention](https://www.terraform.io/docs/modules/index.html#standard-module-structure).